src/test/java/com/twilio/exception/ApiExceptionTest.java

```diff
@@ -29,13 +33,25 @@ public void twoArgConstructorShouldPreserveMessageAndCause() {
     }
 
     @Test
-    public void fullConstructorShouldPreserveAllValues() {
+    public void fiveArgConstructorShouldPreserveAllValues() {
         ApiException error = new ApiException(anyMessage, anyErrorCode, anyMoreInfo, anyHttpStatus, anyCause);
         assertEquals("Message", anyMessage, error.getMessage());
         assertSame("Cause", anyCause, error.getCause());
         assertEquals("More info", anyMoreInfo, error.getMoreInfo());
         assertEquals("Error code", anyErrorCode, error.getCode().intValue());
         assertEquals("Status code", anyHttpStatus, error.getStatusCode().intValue());
+        assertNull("Details", error.getDetails());
+    }
+
+    @Test
+    public void fullConstructorShouldPreserveAllValues() {
+        ApiException error = new ApiException(anyMessage, anyErrorCode, anyMoreInfo, anyHttpStatus, details, anyCause);
+        assertEquals("Message", anyMessage, error.getMessage());
+        assertSame("Cause", anyCause, error.getCause());
+        assertEquals("More info", anyMoreInfo, error.getMoreInfo());
+        assertEquals("Error code", anyErrorCode, error.getCode().intValue());
+        assertEquals("Status code", anyHttpStatus, error.getStatusCode().intValue());
+        assertEquals("Details", error.getDetails(), details);
```

Comment:
nit: `assertEquals("Details", details, error.getDetails());`

Reply:
removed that constructor